### PR TITLE
DPL-588 Traction error messages & Pacbio plate performance

### LINKED ISSUE #1046: DPL-nnn [BUG] Pacbio plates page performance refactor
**Describe the bug**
The pacbio plates page suffers from slow loading speed. We should figure out why this is the case and come up with a solution.

**Note** 
This is most likely due to each plate having its wells and requests mapped when the data is pulled back. This is required so that the 'show plate' functionality works. In order to fix this we could map the plates by an on-demand basis. E.g Only map the plate to its wells and requests when 'show plate' is pressed.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to page 'pacbio plates page'
2. Time to load is significant

**Expected behaviour**
The pacbio plates page should have a similar performance to other pacbio pages.

**Screenshots**

**Desktop (please complete the following information):**
* Windows or Mac?
* Browser Chrome, Firefox, Safari or other?
* Browser version (use 'About' to look up)?

**Additional context**

### PR DESCRIPTION
**Pull request by BenTopping**
Closes #1046

Changes proposed in this pull request:

* Added clear button to remove all messages
* Added debounce to pacbio pool barcode to prevent constant validation warnings
* Removed alert disappear timer
* Added debounce to TractionInput
* Changed pacbio plates page to only pull back plate records and pull back whole plate on 'show plate' click

### CODE REVIEW

**Review by stevieing — COMMENTED** [review]
I think I will check this out and see how it works while reviewing it to provide more context.

**Review by stevieing — DISMISSED** [review]
Good work. I have tested it locally and plates are much faster. All of the other errors seem to be working as expected.
I have added a few advisory comments on things that might be worth doing at some point.

File: tests/e2e/fixtures/pacbioPlatesRequest.json [hunk 1]
```diff
@@ -1,302 +1,72 @@
 {
```
[the remark this answers is not in the document]
Comment (BenTopping) [reply tests/e2e/fixtures/pacbioPlatesRequest.json:1]: Yep I was thinking that as I edited the files. Definitely should be using a message factory or something we could share across e2e and unit for test data.

File: src/store/traction/pacbio/plates/actions.js [hunk 2]
```diff
@@ -2,50 +2,57 @@ import handleResponse from '@/api/ResponseHelper'
...
+  const { success, data: { data } = {}, errors = [] } = response
+
+  if (success) {
+    commit('setPlates', data)
+  }
+
+  return { success, errors }
+}
+
+const getFullPlate = async ({ getters }, filter) => {
```
[the remark this answers is not in the document]
Comment (BenTopping) [reply src/store/traction/pacbio/plates/actions.js:17]: Yeah makes sense, will change it to findPlate.

File: src/store/traction/state.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-const state = {
-  tractionTags: [],
-  messages: {},
+export default () => {
```
[the remark this answers is not in the document]
Comment (BenTopping) [reply src/store/traction/state.js:1]: Making it a method means we can import it into other files and use it as a method call for restoring state to its default. In mutations I use it to restore message state to its default https://github.com/sanger/traction-ui/pull/1056/files#diff-9d2972106a2dc94555c9de66c6572cb004fc8fce33dac26062c5be903a3022e0R26

**Review by stevieing — APPROVED** [review]

**Pull request merged**